Code/GraphMol/Descriptors/EEM.cpp

```diff
@@ -0,0 +1,237 @@
+//
+//  Copyright (c) 2017, Guillaume GODIN
+//  "Copyright 2013-2016 Tomas Racek (tom@krab1k.net)"
```

Comment:
This line worries me. How much of this code is from the original implementation?
That's covered by the GPL and we can't just grab pieces of it and use it.